.github/workflows/integration-test.yml

```diff
@@ -35,17 +36,65 @@ jobs:
               LABELS=("${DEFAULT_LABELS[@]}")
           fi
 
+          # Handle 'core' dependency
           for LABEL in "${LABELS[@]}"; do
             case "$LABEL" in
-              dashboard|consent|pay|core|admin-panel|map|voucher)
+              core|dashboard|consent|pay|admin-panel|map|voucher)
+                BUILD_ARGS+=" //core/api:prod_build"
+                BUILD_ARGS+=" //core/notifications:notifications"
+
+                break
+                ;;
+            esac
+          done
+
+          # Handle root 'node_modules' dependency
+          for LABEL in "${LABELS[@]}"; do
+            case "$LABEL" in
+              dashboard|consent|pay|admin-panel|map|voucher)
+                BUILD_ARGS+=" "//:node_modules""
```

Comment:
same here

Reply:
We only execute line Ln 55 if it's one of the `app` labels right now